README.md

```diff
@@ -125,6 +130,33 @@ The last "unscoped" strategy means there's no file- or
 component-local way of deciding if a style *could* be used elsewhere. You should
 not use sCiSSors on packages / projects with that strategy.
 
+## Using `scissors/css_mirroring_transformer`
+
+Required: Css janus [https://github.com/cegov/wiki/tree/master/maintenance/cssjanus]
+          Note: Two versions available - Google's original cssjanus.py and [https://github.com/cssjanus/cssjanus]. Here the transofrmer uses Google's original cssjanus.py
+          Run the following command in shell:
+            `mkdir -p $HOME/bin && svn checkout http://cssjanus.googlecode.com/svn/trunk/ $HOME/bin && export PATH=$HOME/bin/cssjanus:$PATH`
+
+Example: see [example/mirroring](https://github.com/google/dart-scissors/tree/master/example/mirroring).
+
+`pubspec.yaml`:
+
+  ```
+  dev_dependencies:
+    scissors
+  transformers:
+  - scissors/cssmirroring_transformer
+  ```
+
+Valid settings:
+- `mirrorCss`: `true` by default in release mode and `false` by default in debug mode.
+- `nativeDirection`: Defines the direction of input css. `ltr` by default in debug and release mode.
+- `cssJanusPath`: `cssJanus` by default.
+
+### Limitations
+ - Works only on css files when used as standalone css_mirroring_transformer. 
+ - Does not handle directives like @keyframes and @page.
```

Comment:
please wrap in back-quotes, otherwise this is interpreted as links to github users.
``` `@keyframes` ```, ``` `@page` ```